internal/internal_workflow.go

```diff
@@ -52,7 +52,7 @@ const (
 	defaultCoroutineExitTimeout = 100 * time.Millisecond
 
 	panicIllegalAccessCoroutineState = "getState: illegal access from outside of workflow context"
-	unhandledUpdateWarningMessage    = "Workflow finished while update handlers are still running. This may have interrupted work that the" +
+	unhandledUpdateWarningMessage    = "TMPRL1102 Workflow finished while update handlers are still running. This may have interrupted work that the" +
```

Comment:
```suggestion
	unhandledUpdateWarningMessage    = "[TMPRL1102] Workflow finished while update handlers are still running. This may have interrupted work that the" +
```

Reply:
Right good catch I'll change that in the others